### DISCUSSION TITLE
How to use vant with Vite?

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [youzan/vant#9204] [Q&A] [answered]
I am using `vite`, I followed steps as per 
https://youzan.github.io/vant/v3/#/en-US/quickstart#2.-vite-plugin
 for importing only the components which are used (tree shaking).

But I get below warnings in console and the component is never rendered. 
![image](https://user-images.githubusercontent.com/11911938/128608834-356e51fe-db28-4a84-86f1-68e62261d6ff.png)

**Comment by chenjiahan** [comment, marked as answer]
vant & vite demo project: https://github.com/youzan/vant-demo/tree/master/vant/vite

**Reply by adarshmadrecha** [reply]
As per https://github.com/youzan/vant-demo/blob/69f84ebae2c12d5407f41d31850c7d7df38a2b4a/vant/vite/src/App.vue#L52
We need to import all the components manually for each and every file.

I was hoping for a solution where the imports are taken care automatically (similar to what Vuetify has).
